src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -167,6 +167,11 @@ protected override async Task<bool> AcceptEvent(SessionId sessionId, string meth
                         break;
                     }
 
+                case "Target.targetDestroyed":
```

Comment:
The page is still opened, so the debug target is still there.
When we close the debugger devtools page we receive this message.

Reply:
To test I added this method on program.cs that is called on wasm sample.

```
 [MethodImpl(MethodImplOptions.NoInlining)]
        public static string TestMeaning2()
        {
            return Debugger.IsAttached.ToString();
        }
```

And this method is called by the button that you will see in the video.

Reply:
I think we care about tracking detachment because if we have a dynamic Assembly.Load we don't want to send information to the debugger because it's not attached anymore.

Reply:
When the wasm webpage is closed we receive an Inspector.Detached message and this case I think it doesn't matter.

Reply:
I don't know when we would receive Target.detachedFromTarget, I couldn't find it anywhere.

Reply:
@lewing I will wait your review to merge! :)